geode-core/src/main/java/org/apache/geode/internal/statistics/HostStatSampler.java

```diff
@@ -71,7 +71,7 @@
 
   private static final int WAIT_FOR_SLEEP_INTERVAL = 10;
 
-  private static Thread statThread = null;
+  private Thread statThread = null;
```

Comment:
I think a number of statics are going to cause multiple caches problems.
I think it would be better to file separate tickets on getting rid of those statics instead of lumping them in with this checkin. For example GEODE-4108

Reply:
This and GemFireCacheImpl.instance were the two statics that prevented the tests I wrote from passing.